### ISSUE TITLE
FR:  add Snowflake adapter

### ISSUE THREAD

**Issue by kylelundstedt** [tconbeer/harlequin-web#46]
Hi @tconbeer!  Just started playing with Harlquin; seems very cool.
In addition to DuckDB, we also use Snowflake.  Any chance we'll see an adapter for it?

Thanks!

**Comment by tconbeer** [comment]
Yes, for sure. Any interest in contributing it? Otherwise I may get to it in a while, after a few other features and an odbc adapter. Happy to help you if you want to write it
